apis/user-java/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:8-jdk-alpine
+
+EXPOSE 8080
+
```

Comment:
Missing `ENV` vars required for the datasource* values

Reply:
i thought when you run `docker run` thats where you pass values?

Reply:
Yes, you can pass in or override ENVs via command line but it is best practice to add them into the Dockerfile itself.  This is beneficial because:
- Default values without having to pass them
- Easier to recognize what ENV variables should be present to make the container run